README.md

```diff
@@ -1,87 +1,105 @@
-All the configuration specific to redux
-#### redux.reducers
-The path to the module that exports an **object** of you app reducers.
+```bash
+yarn start
+```
+or if you are not using [yarn](https://yarnpkg.com/)
+```bash
+npm start
+```
+By default, your app will be running on [http://localhost:3000](http://localhost:3000)  
+Build errors will be displayed in your terminal. Everything will be hot reloaded.  
+You can start working on your app immediately !
+
+### Configuration
+VitaminJS contains all of its configuration in a *`.vitaminrc`* file.  
+When you create a new project, it has the minimal configuration to make it work.
 
-vitamin will extend the object with the `react-router-redux` reducer under the key `routing`.That's why it can't be a function created with `combineReducer`.
+If you want to go further in your development, you can customize your configuration. You can learn more about it in [API Guide](https://github.com/Evaneos/vitaminjs/blob/master/API.md)
 
-#### redux.middlewares (optional)
-A path to a module that exports an array of redux middlewares.
+### Build for production
 
-It should look like that:
-```json
-{
-    "routes": "./routes",
-    "redux": {
-        "reducers": "./reducers"
-    }
-}
+To build your application in production mode, run :
+```bash
+yarn build
+```
+or if you are not using [yarn](https://yarnpkg.com/)
+```bash
+npm run build
 ```
 
-Create the `routes.js` file:
+It bundles your application and optimizes its performances. The bundle is also minified.
 
-```js
-import { Route } from 'vitaminjs';
+## Frequently Asked Questions
+We have started to answer some frequently asked questions. You can find all of them here: [FAQ](https://github.com/Evaneos/vitaminjs/blob/master/FAQ.md)  
+If you have any questions you think are worth be displayed, please raise an issue in the project.  
+We will try to update these questions as often as possible. 
 
-export default (
-    <Route path="/">
-);
-```
-Create the `reducers.js` file:
-```js
-export default {};
-```
+## Contributing
 
-And finally you can launch your app with
-```bash
-vitamin start
-```
-Open http://localhost:3000.
+Are you interested in contributing to the project ? Great ! To do so, follow this simple steps:
+ 1. Fork the repository
+ 2. Do the changes you think are necessary. We are following [Angular Commit Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit).
+ 3. Submit a pull request in the project
 
-### Change layout
+## Release History
 
-In your `vitaminrc`:
+You can find all the changes in the [Changelog](https://github.com/Evaneos/vitaminjs/blob/master/CHANGELOG.md)
 
-```json
-{
-    "server": {
-        "layout": "__vitamin__/src/server/components/DivLayout"
-    }
-}
-```
+## Inspiration
+
+[Universal Redux](https://github.com/bdefore/universal-redux)  
+[React-Starter-Kit](https://github.com/kriasoft/react-starter-kit)  
+[Create-React-App](https://github.com/facebookincubator/create-react-app)
```

Comment:
Ok, seems fair. But it's weird to not see it here. Maybe a `similar project` section would be more expected ?